deploy/helm/kubecf/values.yaml

```diff
@@ -366,13 +366,32 @@ k8s-service-username: ""
 k8s-node-ca: ""
 
 eirini:
+  global:
+    labels: {}
+    annotations: {}
+
   env:
     # This setting is not configurable
     # It's a workaround to replace the port eirini uses for the registry
     DOMAIN: '127.0.0.1.nip.io:31666" #'
   services:
     loadbalanced: true
   opi:
+    image_tag:                        ...tag.shared.by.most.images...see.below
+    image:                            ...placeholder...deployment...uses...image_tag
+    loggregator_fluentd_image:        ...placeholder...uses...image_tag
```

Comment:
not required

Reply:
ok

Reply:
Fixed, e82901d6571f1e98ae1912052634b9ba452af906